Gemfile.lock

```diff
@@ -67,7 +76,8 @@ PLATFORMS
 
 DEPENDENCIES
   cocoapods (= 1.1.0.rc.3)
+  slather (= 2.3.0)
   xcpretty (= 0.2.4)
 
 BUNDLED WITH
-   1.13.1
+   1.13.6
```

Comment:
Ah. This just means that you ran `bundle install` with a newer version of bundler than me. I've since updated to 1.13.6 on my machine.

Unfortunately, there's no Gemfile for your Gemfile to manage the version of bundler (at least afaik) people use in your project. You'll likely see changes/diffs like this periodically, and you may end up asking folks to update their bundler or at least revert changes here if this version moves backwards in PRs or something.